README.md

```diff
@@ -3390,6 +3390,16 @@ Other Style Guides
     - We primarily use [`mocha`](https://www.npmjs.com/package/mocha) at Airbnb. [`tape`](https://www.npmjs.com/package/tape) is also used occasionally for small, separate modules.
     - 100% test coverage is a good goal to strive for, even if it’s not always practical to reach it.
     - Whenever you fix a bug, _write a regression test_. A bug fixed without a regression test is almost certainly going to break again in the future.
+    
+  <a name="testing--is-documentation"></a><a name="30.3"></a>
+  - [30.3](#testing--is-documentation) **Is your documentation**:
+    - Tests should explain what your code does.
+    - First write a case for each main functionality, and then their exceptions.
+    - Refactor tests until they are easy to read and understand.
+    - Never use object internal representation from test code, use objects as other code will.
+    - Tests should be copy-paste code to make new developments.
```

Comment:
can you elaborate on what this means?

Reply:
I rewrite it.

Reply:
Ops! Nice found, copypaste ≠ copy-pasteable, it seems that I force people to make copy pastes from here, which it is not my objective.

> Code inside tests should be copy-pasteable code to make new developments.